docs/docs/cmd/spo/orgassetslibrary/orgassetslibrary-add.md

```diff
@@ -13,19 +13,27 @@ m365 spo orgassetslibrary add [options]
 `--libraryUrl <libraryUrl>`
 : The URL of the library to promote.
 
-`--thumbnailUrl <thumbnailUrl>`
+`--thumbnailUrl [thumbnailUrl]`
 : The URL of the thumbnail to render.
 
 `--cdnType [cdnType]`
 : Specifies the CDN type. Allowed values `Public`, `Private`. Default `Private`.
 
+`-o, --orgAssetType [orgAssetType]`
+: Specifies the OrgAssets type. Allowed values `ImageDocumentLibrary`, `OfficeTemplateLibrary`, `OfficeFontLibrary`. Defaults to `ImageDocumentLibrary`.
```

Comment:
`Specifies the OrgAssets type` - is a bit cryptic, as it reads: _orgAssetsType specifies the OrgAssets type_, which doesn't explain anything at all. A clearer explanation would be something like _Specifies the type of organizational library_